CMakeLists.txt

```
@@ -49,6 +49,14 @@ if(dagmc)
   find_package(DAGMC REQUIRED PATH_SUFFIXES lib/cmake)
 endif()
 
+#===============================================================================
+# Check for submodules perhaps already on system
+#===============================================================================
+
+# If not found, we just pull appropriate versions from github and build them.
+find_package(fmt QUIET)
+find_package(pugixml QUIET)
```

```suggestion
find_package(fmt QUIET)
if(fmt_FOUND)
  message(STATUS "Found fmt: ${fmt_DIR} (version ${fmt_VERSION})")
else()
  message(STATUS "Did not find fmt, will use submodule instead")
endif()
find_package(pugixml QUIET)
if(pugixml_FOUND)
  message(STATUS "Found pugixml: ${pugixml_DIR}")
else()
  message(STATUS "Did not find pugixml, will use submodule instead")
endif()
```
In case you're wondering, it looks like fmt provides a version number but pugixml does not

---

Sounds about right. Will get to this soon! Anyways, the revision here looks good.

On the versioning, yeah, I don't think we need to check it because the interfaces for these two libraries seem stable as a rock.